libs/common-services/src/main/java/uk/gov/di/ipv/core/library/dto/RestCriConfig.java

```diff
@@ -12,8 +12,16 @@
 @Getter
 @NoArgsConstructor
 @EqualsAndHashCode(callSuper = true)
-@JsonIgnoreProperties(ignoreUnknown = false)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Making this true makes it easier to add new config values - the values can be set in config and then the code updated to read them

Reply:
Makes sense - not sure why it was false in the first place tbh